source/MRMesh/MRObjectLines.cpp

```diff
@@ -128,133 +82,9 @@ std::vector<std::string> ObjectLines::getInfoLines() const
     return res;
 }
 
-AllVisualizeProperties ObjectLines::getAllVisualizeProperties() const
-{
-    AllVisualizeProperties res;
-    res.resize( LinesVisualizePropertyType::LinesVisualizePropsCount );
-    for ( int i = 0; i < res.size(); ++i )
-        res[i] = getVisualizePropertyMask( unsigned( i ) );
-    return res;
-}
-
-const ViewportMask& ObjectLines::getVisualizePropertyMask( unsigned type ) const
-{
-    switch ( LinesVisualizePropertyType::Type( type ) )
-    {
-    case MR::LinesVisualizePropertyType::Points:
-        return showPoints_;
-    case MR::LinesVisualizePropertyType::Smooth:
-        return smoothConnections_;
-    default:
-        return VisualObject::getVisualizePropertyMask( type );
-    }
-}
-
-ObjectLines::ObjectLines()
-{
-    setDefaultColors_();
-}
-
-void ObjectLines::serializeFields_( Json::Value& root ) const
-{
-    VisualObject::serializeFields_(root);
-
-    root["ShowPoints"] = showPoints_.value();
-    root["SmoothConnections"] = smoothConnections_.value();
-
-    if ( !polyline_ )
-        return;
-    auto& polylineRoot = root["Polyline"];
-    auto& pointsRoot = polylineRoot["Points"];
-    auto& linesRoot = polylineRoot["Lines"];
-    for ( const auto& p : polyline_->points )
-    {
-        Json::Value val;
-        serializeToJson( p, val );
-        pointsRoot.append( val );
-    }
-    for ( UndirectedEdgeId ue{0}; ue < polyline_->topology.undirectedEdgeSize(); ++ue )
-    {
-        auto o = polyline_->topology.org( ue );
-        auto d = polyline_->topology.dest( ue );
-        if ( o && d )
-        {
-            linesRoot.append( int( o ) );
-            linesRoot.append( int( d ) );
-    }
-    }
-
-    // Type
-    root["Type"].append( ObjectLines::TypeName() ); // will be appended in derived calls
-}
```

Comment:
this is important, should not be removed

Reply:
fixed